src/idsk/components/header-extended/header-extended.js

```diff
@@ -138,6 +138,30 @@ HeaderExtended.prototype.checkBlurMenuItemClick = function () {
 HeaderExtended.prototype.showMobilMenu = function (e) {
     this.$module.classList.add("idsk-header-extended--show-mobile-menu")
     document.getElementsByTagName("body")[0].style.overflow = "hidden"
+
+    var $mobileMenuElements = this.$module.querySelectorAll('.idsk-header-extended__mobile')[0].querySelectorAll('a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type="text"]:not([disabled]), input[type="radio"]:not([disabled]), input[type="checkbox"]:not([disabled]), select:not([disabled])');
+    var $firstMenuElement = $mobileMenuElements[0];
+    var $lastMenuElement = $mobileMenuElements[$mobileMenuElements.length-1];
+    var KEYCODE_TAB = 9;
+    document.addEventListener('keydown', function(e) {
```

Comment:
je tu logicka chyba - pretoze pri kazdom otvoreni mobilneho menu, stale pridas dalsi listener.. ak to chces takto, tak by si ho potom mal odoberat pri zavreti mobilneho menu.. pricom ten listener sa ti teraz po prvom otvoreni a zavreti bude spustat aj ked mobilne menu uz nie je otvorene